tests/system/python/packages/test_rule_data_availability.py

```diff
@@ -75,18 +82,44 @@ def start_south(add_south, fledge_url):
     add_south(south_plugin, None, fledge_url, service_name=SOUTH_SERVICE_NAME, installation_type='package', config=config)
 
 
+def _verify_egress(read_data_from_pi_web_api, pi_host, pi_admin, pi_passwd, pi_db, wait_time, retries, asset_name):
+    retry_count = 0
+    data_from_pi = None
+
+    af_hierarchy_level_list = AF_HIERARCHY_LEVEL.split("/")
+    type_id = 1
+    recorded_datapoint = asset_name
+    # Name of asset in the PI server
+    pi_asset_name = asset_name
+
+    while (data_from_pi is None or data_from_pi == []) and retry_count < retries:
+        data_from_pi = read_data_from_pi_web_api(pi_host, pi_admin, pi_passwd, pi_db, af_hierarchy_level_list,
+                                                 pi_asset_name, '')
+        retry_count += 1
+        time.sleep(wait_time * 2)
+
+    if data_from_pi is None or retry_count == retries:
+        assert False, "Failed to read data from PI"
+
 @pytest.fixture
-def start_north(fledge_url, enabled=True):
-    conn = http.client.HTTPConnection(fledge_url)
-    data = {"name": TASK_NAME,
-            "plugin": NORTH_PLUGIN,
-            "type": "north",
-            "enabled": enabled,
-            "config": {"PIServerEndpoint": {"value": "Edge Data Store"},
-                       "NamingScheme": {"value": "Backward compatibility"}}
-            }
-    post_url = "/fledge/service"
-    utils.post_request(fledge_url, post_url, data)
+def start_north(fledge_url, north_historian, start_north_omf_as_a_service, pi_host, pi_port, 
+                pi_admin, pi_passwd, 
+                enabled=True):
```

Comment:
Fix the indentation of the parameters, either grouping them or placing them on a single line, in accordance with PEP 8 guidelines